src/configure/templateInputHelper/InputControlProvider.ts

```diff
@@ -192,7 +192,7 @@ export class InputControlProvider {
 
         for (var clientInput of uniqueDependentClientInputs) {
             var dependentInputControl = this._context[clientInput];
-            if (dependentInputControl) {
+            if (dependentInputControl !== undefined) {
```

Comment:
why this is required.

Reply:
If some value is set to be 'false' in context, this condition evaluates it incorrectly and throws the error that the value doesn't exist in context, though it exists.